fabric-renderer-registries-v1/src/main/java/net/fabricmc/fabric/api/client/rendereregistry/v1/EntityRendererRegistry.java

```diff
@@ -16,6 +16,8 @@
 
 package net.fabricmc.fabric.api.client.rendereregistry.v1;
 
+import net.minecraft.client.render.entity.EntityRenderDispatcher;
+import net.minecraft.client.render.entity.EntityRenderer;
```

Comment:
Extra imports.

Reply:
I think this was the result of resolving some imports so javadoc worked.

`gradlew check` doesn't complain about this